Wikipedia/Code/UIImageView+WMFImageFetchingInternal.m

```diff
@@ -158,12 +158,22 @@ - (void)wmf_setImage:(UIImage *)image
              success:(WMFSuccessHandler)success {
     NSAssert([NSThread isMainThread], @"Interaction with a UIImageView should only happen on the main thread");
 
-    CGRect faceBounds = [faceBoundsValue CGRectValue];
     if (detectFaces) {
-        CGFloat faceArea = faceBounds.size.width * faceBounds.size.height;
-        if (!CGRectIsEmpty(faceBounds) && (faceArea >= 0.05)) {
-            [self wmf_cropContentsByVerticallyCenteringFrame:[image wmf_denormalizeRect:faceBounds]
-                                         insideBoundsOfImage:image];
+
+        CGRect faceBounds = [faceBoundsValue CGRectValue];
+        if (!CGRectIsEmpty(faceBounds)) {
+            
+            CGFloat faceArea = (image.size.width * faceBounds.size.width) * (image.size.height * faceBounds.size.height);
+            CGFloat imageArea = image.size.width * image.size.height;
+            CGFloat faceProportionOfImage = faceArea / MAX(imageArea, 0.0000001);
+            BOOL faceIsBigEnough = (faceProportionOfImage >= 0.01);
```

Comment:
Because the old value wasn't proportional to any particular resolution I had to choose a value that made sense. 

`0.01` of a 640 x 640 would be a 64 x 64 which looked about right:

![screen shot 2017-04-26 at 5 10 25 pm](https://cloud.githubusercontent.com/assets/3143487/25462358/4c80d5ba-2aa3-11e7-8b8c-17d961736837.png)